xcode/Subconscious/Shared/Components/Common/AppThemeBackgroundViewModifier.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AppThemeBackgroundViewModifier.swift
+//  Subconscious
+//
+//  Created by Ben Follington on 5/2/2024.
+//
+
+import SwiftUI
+
+struct AppThemeBackgroundViewModifier: ViewModifier {
+    @Environment(\.colorScheme) var colorScheme
+    
+    func body(content: Content) -> some View {
+        content
+            .background(
+                colorScheme == .dark ? DeckTheme.darkBg : DeckTheme.lightBg
+            )
+            .transition(.identity)
```

Comment:
What does this do?

Reply:
Good catch, turned out not to be needed, I thought it was part of the solution to background jank but it's unrelated.

Removed.